src/game_server/packets/player_update.rs

```diff
@@ -755,7 +810,7 @@ pub struct AddNpc {
     pub image_set_id: u32,
     pub collision: bool,
     pub rider_guid: u64,
-    pub npc_type: u32,
+    pub physics_state: PhysicsState,
```

Comment:
```suggestion
    pub physics: PhysicsState,
```
should we call this setting `physics` over `physics_state`? I think it's okay for the enum to still be called `PhysicsState`, but the user-facing option might be cleaner as simply `physics`